js/index.js

```diff
@@ -40,3 +40,72 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Task One
+const image1 = document.getElementById('cta-img');
+image1.setAttribute("src", siteContent['cta']['img-src'])
+
+const image2 = document.getElementById('middle-img');
+image2.setAttribute("src" , siteContent['main-content']['middle-img-src'])
+
+//Task Two:
+const newLinks = Array.from(document.querySelectorAll('a'));
+let linkName = 'nav-item-';
+let counter =1;
+newLinks.forEach(link => {
+  link.style.color = 'green';
+  link.textContent = siteContent['nav'][linkName + counter++]
+})
```

Comment:
Awesome simplification. This could be even simpler using the `index` that is already built into `.forEach()` like
```
newLinks.forEach((link, i) => {
   link.style.color = 'green';
   link.textContent = siteContent['nav'][i + 1]
 })
```